contrib/build-openbmc.sh

```diff
@@ -25,13 +25,13 @@ meson ../ \
     -Dplugin_ep963x=false \
     -Dplugin_fastboot=false \
     -Dplugin_logitech_bulkcontroller=false \
-    -Dplugin_nitrokey=false \
-    -Dplugin_nvme=false \
+    -Dplugin_nitrokey=disabled \
+    -Dplugin_nvme=disabled \
```

Comment:
it does make me a bit uncomfortable having `disabled` and `false` but i see you're busy converting some others too. Keep going and we'll see what it looks like later.

Reply:
Before I go too much further - can you let me know what you're thinking of this and what it does?